WORKSPACE

```diff
@@ -1,10 +1,11 @@
 workspace(name = "io_bazel_rules_scala")
 
-load("//scala:scala.bzl", "scala_repositories", "scala_mvn_artifact")
+load("//scala:scala.bzl", "scala_repositories", "scala_mvn_artifact", "scrooge")
 scala_repositories()
+scrooge()
```

Comment:
should probably call this `scala_scrooge` or `twitter_scrooge` or something a bit more namespaced.